src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
Yes, and the correct latex command is `$\Epsilon$` without any braces! The reason I'm insisting on this is that BibLatex/Biber can automatically convert Latex code to Unicode, but then of course `{{\"a}}` is converted to `{ä}` which is not the desired output.

By the way, it is better to preserve the capitalization of `Ägypten` by `{{\"{A}}gypten}` (i.e. the braces enclose the whole word and not just the capital letter so that kerning works as expected), see http://tex.stackexchange.com/a/140071/34554. If the preserve capitalization cleanup is not able to do this right now, then it should be improved...

Finally, have a look at https://github.com/plk/biber/blob/dev/lib/Biber/LaTeX/recode_data.xml which provides a huge map from latex to unicode (and I like the data scheme more then our internal maps!). Finally, https://github.com/plk/biber/blob/dev/lib/Biber/LaTeX/Recode.pm may also be helpful.